libs/langgraph/src/managed/context.ts

```diff
@@ -0,0 +1,39 @@
+import { EmptyChannelError } from "../errors.js";
+import { ManagedValue, ConfiguredManagedValue } from "./base.js";
+
+// Type for a synchronous context manager
+export type ContextManager<T> = {
+    enter: () => T;
```

Comment:
Can you have multiple values that need to be exposed to `exit()`?

Wondering if it actually would be cleaner to use an iterator...